src/EnergyPlus/EvaporativeCoolers.hh

```diff
@@ -401,9 +401,9 @@ namespace EvaporativeCoolers {
                               Real64 const FlowRatio   // secondary air flow fraction
     );
 
-    void CalcIndirectResearchSpecialEvapCooler(int const EvapCoolNum);
+    void CalcIndirectResearchSpecialEvapCooler(int const EvapCoolNum, Real64 const FanPLR);
 
-    void CalcDirectResearchSpecialEvapCooler(int const EvapCoolNum);
+    void CalcDirectResearchSpecialEvapCooler(int const EvapCoolNum, Real64 const FanPLR);
```

Comment:
The unit tests are now failing since the FanPLR is a required argument. You also added a 1.0 at line 2165 above so that the code would compile. Look at [this default argument](https://github.com/NREL/EnergyPlus/pull/7230#issuecomment-517287379) declaration to see if you would rather go that route.

Reply:
Done, thanks!